tests/test_commands.py

```diff
@@ -84,3 +88,20 @@ def test_invalid_field_path(self):
         error_message = "Error parsing field_path 'MyStorageModel.image'. "\
                         "Use format <app.model.field app.model.field>."
         assert str(exc_info.value) == error_message
+
+    def test_custom_render_variations(self, image_upload_file):
+        obj = CustomRenderVariationsModel.objects.create(
+            image=image_upload_file
+        )
+        file_path = obj.image.thumbnail.path
+        assert os.path.exists(file_path)
+
+        before = hashlib.md5(open(file_path, 'rb').read()).hexdigest()
```

Comment:
since py.test doesn't run in boxed mode you should close all files in each test to reduce test load.